.github/workflows/codecoverage.yml

```diff
@@ -1,5 +1,5 @@
 ---
-name: code_coverage
+name: "Code Coverage 🎯"
```

Comment:
why are we adding emoticons here?

Reply:
so that it looks fun and not boring.
tbh, makes it easier to read from a bunch of CI jobs.